pkg/staticnetworkconfig/generator_test.go

```diff
@@ -284,9 +306,104 @@ var _ = Describe("validate mac interface mapping", func() {
 					},
 					NetworkYaml: withoutUnderlyingInterface,
 				},
-			})
+			}
+			err := validateStaticConfigMethod(staticNetworkConfig)
 			Expect(err).ToNot(HaveOccurred())
-		})
+		},
+			table.Entry("ocp versions less than 4.13", staticNetworkGenerator.ValidateStaticConfigParams),
+			table.Entry("ocp versions greater than/ equal to 4.13", staticNetworkGenerator.ValidateStaticConfigParamsYAML),
+		)
+	})
+
+	Context("no mapping for physical interfaces", func() {
+		withPhysicalInterface := `interfaces:
+  - name: eth0
+    type: ethernet
+    state: up
+    ipv4:
+      enabled: true
+      dhcp: false
+      address:
+        - ip: 192.0.2.1
+          prefix-length: 24
+  - name: eno12399np0
+    type: ethernet
+    state: up
+    ipv4:
+      enabled: false
+      dhcp: false`
+		withNoMappedInterfaces := `interfaces:
+  - name: eno12345
+    type: ethernet
+    state: up
+    ipv4:
+      enabled: false
+      dhcp: false
+  - name: eno12399np0
+    type: ethernet
+    state: up
+    ipv4:
+      enabled: false
+      dhcp: false`
+		withMacIdentifier := `interfaces:
+  - name: eth0
+    type: ethernet
+    state: up
+    identifier: mac-address
+    mac-address: f8:75:a4:a4:00:fe
+    ipv4:
+      enabled: false
+      dhcp: false`
+		table.DescribeTable("no mapping needed for physical interface", func(validateStaticConfigMethod func([]*models.HostStaticNetworkConfig) error) {
+			staticNetworkConfig := []*models.HostStaticNetworkConfig{
+				{
+					MacInterfaceMap: []*models.MacInterfaceMapItems0{
+						{
+							LogicalNicName: "eth0",
+							MacAddress:     "f8:75:a4:a4:00:fe",
+						},
+					},
+					NetworkYaml: withPhysicalInterface,
+				},
+			}
+			err := validateStaticConfigMethod(staticNetworkConfig)
+			Expect(err).ToNot(HaveOccurred())
+		},
+			//table.Entry("ocp versions less than 4.13", staticNetworkGenerator.ValidateStaticConfigParams),
```

Comment:
Is this needed or not? If not let's get rid of the `DescribeTable` when there's only one entry.